data/transactions/logic/assembler.go

```diff
@@ -2411,8 +2414,19 @@ func (ops *OpStream) warnf(format string, a ...interface{}) error {
 	return ops.warn(fmt.Errorf(format, a...))
 }
 
-// ReportProblems issues accumulated warnings and outputs errors to an io.Writer.
-func (ops *OpStream) ReportProblems(fname string, writer io.Writer) {
+// MultipleErrors issues accumulated warnings and outputs errors to an io.Writer.
+// In the case of exactly 1 error and no warnings, a slightly different format is provided
+// to handle the cases when the original error is or isn't reported elsewhere.
+// In the case of > 10 errors, only the first 10 errors will be reported.
+func (ops *OpStream) MultipleErrors(fname string, writer io.Writer) {
```

Comment:
`MultipleErrors` name is misleading, `ReportMultipleErrors` or just `ReportProblems` imo is better

Reply:
Renamed to `ReportMultipleErrors` 5f7de10984475128b8d2ec5871e10318f844fd8e